src/components/VideoTile.tsx

```diff
@@ -5,6 +5,7 @@ import { AssetAvailability } from '@/api/queries'
 import { absoluteRoutes } from '@/config/routes'
 import { AssetType, useAsset } from '@/providers/assets'
 import { singleDraftSelector, useDraftStore } from '@/providers/drafts'
+import { useUploadsStore } from '@/providers/uploadsManager'
```

Comment:
This import will cause the UploadsManager to be moved into the main chunk instead of the studio one as it is currently. I think if you split this file into `VideoTile` and `VideoTilePublisher`, we should be good